test/regression/ode_adaptive_tests.jl

```diff
@@ -125,3 +125,22 @@ sol_linear = solve(prob_linear, ESDIRK547L2SA2())
 
 sol_lorenz = solve(prob_lorenz, ESDIRK547L2SA2())
 @test length(sol_lorenz.u) < 1000
+
+# Adaptivity tests for Alshina2, 3
+
+function lorenz!(du,u,p,t)
+    du[1] = 10.0(u[2]-u[1])
+    du[2] = u[1]*(28.0-u[3]) - u[2]
+    du[3] = u[1]*u[2] - (8/3)*u[3]
+end
+u0 = [1.0;0.0;0.0]
+tspan = (0.0,1.0)
+prob_lorenz = ODEProblem(lorenz!,u0,tspan)
+
+sol_linear = solve(prob_linear, Alshina2())
+sol_lorenz = solve(prob_lorenz, Alshina2())
```

Comment:
and 6?

Reply:
The embedded schemes were not defined for ERK6. [Reference](https://link.springer.com/article/10.1134/S0965542508030068)